specs/interop/derivation.md

```diff
@@ -123,10 +122,3 @@ When fully executing historical blocks, a dependency on historical receipts from
 needing to execute increasingly long chain histories.
 
 [eip-4444]: https://eips.ethereum.org/EIPS/eip-4444
-
-### Expiry Window
-
-The expiry window ensures that the proof can execute in a reasonable amount of time.
-There is currently no way to prove old history with a sublinear proof size. The proof
-program needs to walk back and re-execute to reproduce the consumed logs. This means
-that very old logs are more expensive to prove.
```

Comment:
can we keep this section and instead write
> App devs can rely on
https://github.com/ethereum-optimism/design-docs/blob/25ef5537e39b63cddf1c83479cee9f0e02431dce/protocol/resend-messages.md#L4
to get around this issue

from the pr description in this section?

Reply:
agree. https://github.com/ethereum-optimism/specs/pull/681